test_unstructured/documents/structured_jsons/example.json

```diff
@@ -0,0 +1,121 @@
+[
+    {
+        "element_id": "897a8a47-377c-4ad6-aab8-39a929879537",
+        "metadata": {
+            "category_depth": 0,
+            "text_as_html": "<body class=\"Document\" id=\"897a8a47-377c-4ad6-aab8-39a929879537\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+        "metadata": {
+            "category_depth": 1,
+            "page_number": 1,
+            "parent_id": "897a8a47-377c-4ad6-aab8-39a929879537",
+            "text_as_html": "<div class=\"Page\" data-page-number=\"1\" id=\"3a6b156a-8176-4e17-be12-8264241f8136\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "45b3d005-3468-484b-a1c7-b53998115412",
+        "metadata": {
+            "category_depth": 2,
+            "page_number": 1,
+            "parent_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+            "text_as_html": "<header class=\"Header\" id=\"45b3d005-3468-484b-a1c7-b53998115412\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "c95473e8-a370-4fc2-b418-697f9fddb27b",
+        "metadata": {
+            "category_depth": 3,
+            "page_number": 1,
+            "parent_id": "45b3d005-3468-484b-a1c7-b53998115412",
+            "text_as_html": "<h1 class=\"Title\" id=\"c95473e8-a370-4fc2-b418-697f9fddb27b\">Header </h1>"
+        },
+        "text": "Header",
+        "type": "Title"
+    },
+    {
+        "element_id": "379cbfdc-16d4-4bd6-a59e-6cfabe6438d5",
+        "metadata": {
+            "category_depth": 3,
```

Comment:
just noting that this has a`"category_depth": 3,` which is strictly true as UncategorizedText, though maybe not particularly useful.